### ISSUE TITLE
Unstable examples in codesandbox 

### ISSUE THREAD

**Issue by flyingonionman** [pmndrs/lamina#26]
<img width="735" alt="Screen Shot 2022-04-11 at 5 05 14 PM" src="https://user-images.githubusercontent.com/39283458/162832049-5bf373e4-e6e1-41f1-87b8-f726798dc303.png">

https://codesandbox.io/embed/github/pmndrs/lamina/tree/main/examples/layer-materials

Is this issue related to code-sandbox? 

The [complex-materials](https://codesandbox.io/embed/github/pmndrs/lamina/tree/main/examples/complex-materials) example works, which leads me to believe that maybe its webpack related

**Comment by FarazzShaikh** [comment]
Yes thats a code-sandbox issue. It does not like custom configurations. 

The examples will be converted to CRA template with the next release.
